test/Microsoft.SqlTools.ServiceLayer.UnitTests/EditData/RowDeleteTests.cs

```diff
@@ -193,7 +193,7 @@ public async Task RevertCell()
             Assert.Throws<InvalidOperationException>(() => rd.RevertCell(0));
```

Comment:
Is this required? Do we need changes in SqlTools.ServiceLayer?

Reply:
I had to make that change for the branch to build. I refreshed from feature-ADE and it looks like it got resolved.